client.go

```diff
@@ -768,22 +783,37 @@ func (c *TunaSessionClient) handleMsg(conn *Conn, sess *ncp.Session, i int) erro
 	return nil
 }
 
-func (c *TunaSessionClient) handleConn(conn *Conn, sessKey string, i int) {
+func (c *TunaSessionClient) handleConn(conn *Conn, remoteAddr string, sessionID []byte, i int) {
 	if conn == nil {
 		return
 	}
+
+	sessKey := sessionKey(remoteAddr, sessionID)
 	c.Lock()
 	sess := c.sessions[sessKey]
 	if sess == nil {
 		c.Unlock()
 		return
 	}
 	c.connCount[sessKey]++
+	tcpCount, ok := c.addrConnCount[remoteAddr]
+	if ok {
+		tcpCount[conn.RemoteAddr().String()]++
```

Comment:
`conn.RemoteAddr().String()` is computed a lot of times here

Reply:
Yes, modified.